ui/src/application/Applications.tsx

```diff
@@ -168,6 +173,14 @@ const Row: SFC<IRowProps> = observer(
             <TableCell>
                 <ToggleVisibility value={value} style={{display: 'flex', alignItems: 'center'}} />
             </TableCell>
+            <TableCell>
+                <IconButton
```

Comment:
This code is pretty much the same on the other components, so would be great create a separate component(`CopyToClipboardButton`) for it inside `src/common` directory.

Reply:
Thank you very much!
@jmattheis also suggested injecting snackManager into a component. But I didn't understand what they meant until I saw your example.
I'm going to apply the changes you suggested asap, cheers!